mnefun/_mnefun.py

```diff
@@ -1678,22 +1715,7 @@ class FakeEpochs():
     """Make iterable epoch-like class, convenient for MATLAB transition"""
 
     def __init__(self, data, ch_names, tmin=-0.2, sfreq=1000.0):
-        self._data = data
-        self.info = dict(ch_names=ch_names, sfreq=sfreq)
-        self.times = np.arange(data.shape[-1]) / sfreq + tmin
-        self._current = 0
-        self.ch_names = ch_names
-
-    def __iter__(self):
-        self._current = 0
-        return self
-
-    def next(self):
-        if self._current >= len(self._data):
-            raise StopIteration
-        epoch = self._data[self._current]
-        self._current += 1
-        return epoch
+        raise RuntimeError('Use mne.EpochsArray instead')
 
```

Comment:
I don't understand this. It's a class function that does what beside `raise RuntimeError`?